systemd/ceph-mgr@.service.in

```diff
@@ -12,7 +12,7 @@ Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-mgr -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecReload=/bin/kill -HUP $MAINPID
 LockPersonality=true
-MemoryDenyWriteExecute=true
+MemoryDenyWriteExecute=false
```

Comment:
Please add a comment that this is needed for FFI/JIT.

Reply:
done